### DISCUSSION TITLE
bbb

### DISCUSSION THREAD

**Discussion by FedeGueli** [ciscorucinski/SARS-CoV-2_Variant_Designation_Tree#1] [Proposal]
bbb https://nextstrain.org/fetch/genome.ucsc.edu/tra
![Schermata 2023-05-09 alle 15 39 44](https://github.com/ciscorucinski/SARS-CoV-2_Variant_Designation_Tree/assets/87669813/920ab6ac-2118-4598-93a3-090bcf5d1646)
sh/ct/

**Comment by ciscorucinski** [comment]
Move to Proposal Category.

The following information is needed to complete this proposal
- [x] Info 1
- [x] Info 2
- [x] Info 3
- [x] Info 4
- [ ] Info 5

**Comment by ciscorucinski** [comment]
Although Info 5 is missing, we'll move forward with this proposal
